### DISCUSSION TITLE
I/O Error 105.

### DISCUSSION THREAD

**Discussion by kazuser** [SchneiderInfosystems/FB4D#103] [General]
![изображение](https://user-images.githubusercontent.com/3335273/122630466-b7858080-d0e5-11eb-929b-565a408acde6.png)
I faced with something strange on fresh installed Windows 10 😶

1) Download current version

2) Open and compile \Samples\Intro\

3) Fill the "Web API Key" field and press the "Anonymous Login" button:
![изображение](https://user-images.githubusercontent.com/3335273/122630484-ee5b9680-d0e5-11eb-809b-9760d9bbbe50.png)

4) After closing:
![изображение](https://user-images.githubusercontent.com/3335273/122630485-f3204a80-d0e5-11eb-8a3e-8955ad783159.png)

Did I do something wrong? The paths are fine 🤔

**Comment by kazuser** [comment]
![изображение](https://user-images.githubusercontent.com/3335273/122630781-2a8ff680-d0e8-11eb-95ad-d305fe9f4bf1.png)

I've found in my old Downloads folder previous .zip-file and it works fine but I would update it :)
![изображение](https://user-images.githubusercontent.com/3335273/122630737-dedd4d00-d0e7-11eb-858a-1272ff6a78c0.png)

**Reply by SchneiderInfosystems** [reply]
As explained in the wiki you have to install OpenSSL first when using a project with TOKENJWT symbol as conditional compiling. If you do not need in your project to verify the JWTToken from Auth you do not need this optional feature. Please read here:
https://github.com/SchneiderInfosystems/FB4D/wiki/Getting-Started-with-FB4D#install-openssl

**Comment by eag1980** [comment]
I had the same problem. I solved the problem by commenting out line 354 of the FB4D.Helpers file.

It looked like this:

class procedure TFirebaseHelpers.Log(msg: string);
begin
   if AppIsTerminated then
     exit;
{$IF Defined(FMX)}
   {$IFDEF LINUX}
   writeln(msg); // Workaround for RSP-32303
   {$ELSE}
   FMX.Types.Log.d(msg, []);
   // is there a bug in DE 10.2 when the wrong method is calling?
   {$ENDIF}
{$ELSEIF Defined(VCL)}
   OutputDebugString(PChar(msg));
{$ELSE}
// writeln(msg); ******************************
{$ENDIF}
end;

**Reply by SchneiderInfosystems** [reply]
I/O error 105 occurs if you compile a project without defining the symbol $(FrameworkType) as compiler conditional. For more information please read this wiki page:
https://github.com/SchneiderInfosystems/FB4D/wiki/Project-settings-for-FB4D

**Reply by kazuser** [reply]
My bad. Thanks for the info!
